### ISSUE TITLE
[Stupid Question] Is Volatile Dead's DPS being calculated correctly?

### ISSUE THREAD

**Issue by Thirsk** [Openarl/PathOfBuilding#780]
It seems as though Volatile Dead's displayed DPS is just multiplying the average hit by the cast speed, but Volatile Dead creates at least three orbs per cast. Are the three separate hits accounted for already in the average hit in some way I'm not seeing, or is Volatile Dead's displayed DPS only a third of what it actually is?

**Comment by master2080** [comment]
It's probably just based on a single projectile, as for many skills (like tornado shot).

**Comment by Thirsk** [comment]
Fair enough, but with Tornado Shot it's unpredictable how many will hit while Volatile Dead will always hit 3 times due to the fact that the orbs seek out enemies (unless you can't sustain corpses, but it seems reasonable to assume that you'll be able to do that).

**Comment by Asday** [comment]
The orbs may seek out different targets.

If you know for a fact you'll be hitting the same mob with all of your balls, simply multiply the number you see by the number of balls you have in your head.

That being said, I did notice that the DPS listed _felt_ a little low compared to how I stacked up against other builds ingame.
